.travis.yml

```diff
@@ -23,16 +23,17 @@ jobs:
     env:
     - CIBUILDWHEEL=1
     - CIBW_BUILD="cp*-manylinux_aarch64"
-    - CIBW_MANYLINUX_AARCH64_IMAGE=manylinux2014
     - CIBW_ENVIRONMENT_PASS_LINUX="GEOS_VERSION GEOS_INSTALL GEOS_CONFIG LD_LIBRARY_PATH"
     - CIBW_BEFORE_ALL="./ci/install_geos.sh"
     - CIBW_TEST_REQUIRES="pytest"
-    - CIBW_TEST_COMMAND="pytest --pyargs pygeos.tests"
+    - CIBW_TEST_COMMAND="pytest --pyargs shapely.tests"
+
 
 branches:
   only:
-  - master
+  - main
   - travis
+  - /^\d+\.\d+(\.\d+)?(-\S*)?$/
```

Comment:
According to https://docs.travis-ci.com/user/conditions-v1, we should be able to add something like `if: tag IS present` for having it only run on tags (on main)